tasks/audit_system_call.yml

```diff
@@ -1,7 +1,7 @@
 ---
 # tasks/audit_system_call.yml
 
-- name: "MEDIUM | RHEL-07-{{ item.id }} | {{ audit_present | ternary('PATCH', 'REVERT') }} | All uses of the {{ item.call }} command must be audited."
+- name: "MEDIUM | RHEL-07-{{ item.id }} | {{ audit_present | ternary('PATCH', 'REVERT') }} | The Red Hat Enterprise Linux operating system must audit all uses of the  {{ item.call }} syscall."
```

Comment:
Are there two spaces in a row in the DISA content?

Reply:
Talk about "eagle-eyed"...

I just pushed an update to correct the extra space!  Should be good to go!